packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -39,6 +39,11 @@ export interface IButtonProps extends React.HTMLProps<HTMLButtonElement | HTMLAn
   ariaDescription?: string;
 
   /**
+  * Text to render button label. If text is supplied, it will override any string in button children. Other children components will be passed through after the text.
+  */
+  text?: string;
```

Comment:
So the `label` prop that was in use before was from HTMLButtonElement, and not a prop we documented?